src/index.ts

```diff
@@ -0,0 +1,173 @@
+import "dotenv/config";
+import express, { Request, Response } from "express";
+import cors from "cors";
+import { drizzle } from "drizzle-orm/libsql";
+import { usersTable } from "./db/schema.js";
+import { randomBytes } from "node:crypto";
+import { eq } from "drizzle-orm";
+import { LibsqlError } from "@libsql/client";
+
+const db = drizzle(process.env.DB_FILE_NAME!);
```

Comment:
I think this should belong to your `db` folder to keep all database related setups, schemas, helpers, ... at the same place